lib/taskana-core/src/main/java/pro/taskana/TaskanaConfiguration.java

```diff
@@ -743,6 +757,11 @@ public static class Builder {
     private Set<WorkbasketPermission> minimalPermissionsToAssignDomains = new HashSet<>();
     // endregion
 
+    // region database configuration
+    @TaskanaProperty("taskana.feature.useSpecificDb2Taskquery")
+    private boolean useSpecificDb2Taskquery = false;
```

Comment:
Didn't we say the default is the old special DB2 query ? If yes, shouldn't the default be true?

Reply:
Yes, I inverted all values if this parameter.